docs/4.1/guides/installation.md

```diff
@@ -0,0 +1,146 @@
+# Installation
+
+This page will show you how to install Teleport. Teleport is available for **Linux, Mac**, and **Windows** operating systems. Our examples install teleport v4.1.0 but you can install any version listed in our [Release History](https://gravitational.com/teleport/releases/).
```

Comment:
For this I would make it clear that TSH is available for **Linux, Mac**, and **Windows** operating systems, and Teleport has been designed to work on Linux / Macs.